core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/Ellipse2afp.java

```diff
@@ -56,6 +56,11 @@
 		B extends Rectangle2afp<?, ?, IE, P, V, B>>
 		extends RectangularShape2afp<ST, IT, IE, P, V, B> {
 
+    final String ELLIPSE_WIDTH_POSITIVE_OR_ZERO = "Ellipse width must be positive or zero";
```

Comment:
I will check where the code does not fit the Checkstyle plugin.  As for the
new class ( AssertionMessages)  containing all the messages for the said
relating classes seems to be a good idea. Please allow me to do it my style
first, then I will attempt to bring them within a class in a convenient
time within this week. Because I am running against time and need to create
many PRs everyday for my rating.

On Sun, May 29, 2016 at 1:27 AM, Stéphane Galland <notifications@github.com>
wrote:

> In
> core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/Ellipse2afp.java
> <https://github.com/gallandarakhneorg/afc/pull/52#discussion_r64995100>:
>
> > @@ -56,6 +56,11 @@
> >  		B extends Rectangle2afp<?, ?, IE, P, V, B>>
> >  		extends RectangularShape2afp<ST, IT, IE, P, V, B> {
> >
> > +    final String ELLIPSE_WIDTH_POSITIVE_OR_ZERO = "Ellipse width must be positive or zero";
>
> The errors seems to be given by the Checkstyle plugin that was now used
> for validating the code. Please fix the code according to the
> recommendations of Checkstyle, in addition to remove the IntelliJ
> configuration files.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/gallandarakhneorg/afc/pull/52/files/c8d39b6497aeb251c828ba1a5d33b33234c7a750#r64995100>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AR09BJyqdl3sw5DizdZlg-c2lF9OuSNpks5qGMFGgaJpZM4IpKH_>
> .
>